src/util/dump.ts

```diff
@@ -42,7 +43,10 @@ export async function getAndProcessStylesheets(downloader: Downloader, links: Ar
                     if (!url.match('^data')) {
                         const filePathname = urlParser.parse(url, false, true).pathname;
                         if (filePathname) {
-                            const filename = pathParser.basename(filePathname).replace(/-.*x./, '.');
+                            const filename = pathParser.basename(filePathname).replace(/-.*x./, '.')
+                            + shouldConvertImageFilenameToWebp(pathParser.basename(filePathname).replace(/-.*x./, '.'), downloader.webp)
```

Comment:
@Jaifroid Sorry in advance for the bit silly question, but do you know if the polyfill will apply ton `webp` pictures used in CSS?

Reply:
@MananJethwani Like faired, the polyfill can not apply to the pictures in the CSS. We have to secure we don't apply webp optimisation to the pictures the CSS relies on. I don't know exactly how, but considering that these pictures have to be saved in an other namespace, we could try to rely on this to make the filtering.